client/js/options.js

```diff
@@ -300,8 +312,12 @@ function initialize() {
 		}
 	});
 
+	$settings.on("click", "#forceSync", () => {
```

Comment:
Should this enable sync?

Reply:
I don't think so, now it has a single clear purpose. I personally wouldn't expect such a button to also enable sync.

And there might be use cases where people just want to force sync the settings of one client without enabling sync.

I also don't really expect buttons in an interface with check boxes to enable anything (Push notifications being a button seems odd to me for example). 

I thought about doing two checkboxes. One for enabling sync and one for force syncing before enabling sync but that seems confusing and easier to overlook than a big green button saying you can force sync. 

In short, this is the cleanest clearest solution in my opinion.

Reply:
Okay this is fine by me, I'll let other decide on this.

However can you change this line to read `$settings.find("#forceSync").on("click", () => {`

Reply:
Done